frontend/src/composables/TabPanels/useChildsTabData.js

```diff
@@ -0,0 +1,141 @@
+import { ref } from "vue";
+import { useStore } from "vuex";
+import { useRouter } from "vue-router";
+import { useQuasar } from "quasar";
+
+export default function useChildsTabData(props) {
```

Comment:
Rename `child` to `workspace`. Because child is not relevant, do this for all child in this pull request.

Reply:
Ok